data/consent-page.html

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+
+<!DOCTYPE html>
+  <html xmlns="http://www.w3.org/1999/xhtml hasBrowserHandlers="true">
+    <head>
+      <meta charset="utf-8" />
+      <title>Participate in SHIELD Studies</title>
+      <link rel="stylesheet" href="consent-page.css" type="text/css" />
+    </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title studyName"></h1>
+          <button id="installAddon" class="addonUrl button primary"></button>
+        </div>
+        <div class="content">
+          <h2 id="what-will-happen-next">What will happen next</h2>
```

Comment:
Ah, this was just lifted from glinds consent page template creator. Hadn't looked much into the HTML of it.